packages/Pager/src/Pager.tsx

```diff
@@ -153,7 +159,7 @@ const Pager: React.FunctionComponent<PagerProps> = ({
             </nav>
             <div className="ids-pager__results">
                 {pageSize * currentPage - pageSize + 1}-
-                {Math.min(pageSize * currentPage, totalCount)} of {totalCount}
+                {Math.min(pageSize * currentPage, totalCount)} {stringFormatter.format("of")} {totalCount}
```

Comment:
i'm 90% certain that there is a way for the stringFormatter to be used with templates. Looking at react-spectrum, it should be 

in intl file:
```
  "pageRange": "{count} of {totalCount}"
```

in js
```
stringFormatter.format('pageRange', {count: 1, totalCount: 4});
```

Reply:
same comment for                      

                          : `${stringFormatter.format("goToPage")} ${pageNumber}`

Reply:
yes it can be, but I figured they were positioned in the same place so I didn't add it. If you think I should, then I'll fix it.

Reply:
We should do it the right way ya. to start getting used to it